Failed to build c++ client at first time in linux

**Describe the bug**
Following the [doc ](http://pulsar.apache.org/docs/en/next/develop-cpp/), someone may encounter the below building error:
```
root@hdp1:/opt/project/pulsar/pulsar-client-cpp# make
[  1%] Generating PulsarApi.pb.h, PulsarApi.pb.cc
/bin/sh: 1: protoc: not found
lib/CMakeFiles/pulsarShared.dir/build.make:61: recipe for target 'lib/PulsarApi.pb.h' failed
make[2]: *** [lib/PulsarApi.pb.h] Error 127
CMakeFiles/Makefile2:151: recipe for target 'lib/CMakeFiles/pulsarShared.dir/all' failed
make[1]: *** [lib/CMakeFiles/pulsarShared.dir/all] Error 2
Makefile:127: recipe for target 'all' failed
make: *** [all] Error 2
```

**Desktop (please complete the following information):**
 - OS:ubuntu16.04

--- a/site2/docs/developing-cpp.md
+++ b/site2/docs/developing-cpp.md
@@ -34,7 +34,7 @@ First, install all of the necessary dependencies:
 
 ```shell
 $ apt-get install cmake libssl-dev libcurl4-openssl-dev liblog4cxx-dev \
-  libprotobuf-dev libboost-all-dev google-mock libgtest-dev libjsoncpp-dev
+  libprotobuf-dev protobuf-compiler libboost-all-dev google-mock libgtest-dev libjsoncpp-dev
 ```
 
 Then compile and install [Google Test](https://github.com/google/googletest):

